guides/source/action_cable_overview.md

```diff
@@ -357,6 +365,64 @@ consumer.subscriptions.create({ channel: "ChatChannel", room: "1st Room" })
 consumer.subscriptions.create({ channel: "ChatChannel", room: "2nd Room" })
 ```
 
+Rails provides a generator to scaffold the server and client-side channel files:
+
+```bash
+$ bin/rails generate channel
+Usage:
+  rails generate channel NAME [method method] [options]
+
+...
+
+Example:
+========
+    bin/rails generate channel Chat speak
+
+    creates a Chat channel class, test and JavaScript asset:
+        Channel:    app/channels/chat_channel.rb
+        Test:       test/channels/chat_channel_test.rb
+        Assets:     $JAVASCRIPT_PATH/channels/chat_channel.js
+```
+
+#### Importing channels
```

Comment:
Headings in the guides should have all words capitalized, except for prepositions, conjunctions, internal articles, and forms of the verb "to be":
https://guides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#headings
```suggestion
#### Importing Channels
```

Reply:
Oops, thanks for catching this, @p8 ! I've updated the heading now.